test/common/testlib.py

```diff
@@ -1417,7 +1417,7 @@ def wait(func, msg=None, delay=1, tries=60):
             val = func()
             if val:
                 return val
-        except:
+        except Error:
```

Comment:
I think this is why so many tests fail -- we also often use this function to wait for a `Machine.execute()` to succeed. Thus the tests now fail on `subprocess.CalledProcessError`. Maybe try with `(Error, subprocess.SubprocessError)` for now, and if that still fails, we need to generalize it further?